solr/core/src/test/org/apache/solr/response/TestSolrQueryResponse.java

```diff
@@ -90,35 +90,30 @@ public void testResponse() throws Exception {
   public void testToLog() throws Exception {
     final SolrQueryResponse response = new SolrQueryResponse();
     assertEquals("toLog initially not empty", 0, response.getToLog().size());
-    assertEquals("logid_only", response.getToLogAsString("logid_only"));
+    assertEquals("{\"prefix\":\"prefix_only\"}", response.getToLogAsString("prefix_only"));
     // initially empty, then add something
     response.addToLog("key1", "value1");
-    {
-      final Iterator<Map.Entry<String, Object>> it = response.getToLog().iterator();
-      assertTrue(it.hasNext());
-      final Map.Entry<String, Object> entry1 = it.next();
-      assertEquals("key1", entry1.getKey());
-      assertEquals("value1", entry1.getValue());
-      assertFalse(it.hasNext());
-    }
-    assertEquals("key1=value1", response.getToLogAsString(""));
-    assertEquals("abc123 key1=value1", response.getToLogAsString("abc123"));
+
+    Map<String, Object> toLog = response.getToLog();
```

Comment:
I recommend doing assertEquals with a Map you create yourself like {{Map.of("key1", "value1")}}.  Same with further below -- way more concise and easy to read/review.